src/main/java/TaskStatus.java

```diff
@@ -0,0 +1,16 @@
+public enum TaskStatus {
+    DONE("X"),
+    UNDONE(" ");
+
+    private String StatusSymbol;
+
+    TaskStatus(String StatusSymbol) {
+        this.StatusSymbol = StatusSymbol;
+    }
+
+    public String getStatusSymbol() {
```

Comment:
Consider adding a Javadoc comment for this. Specifically, what is the status that is returned about.